src/Parser/MistweaverMonk/Modules/Items/Velens.js

```diff
@@ -34,7 +34,25 @@ class Velens extends Module {
       return;
     }
 
-    this.healing += calculateEffectiveHealing(event, LEGENDARY_VELENS_HEALING_INCREASE);
+    // Rewriting the effectiveHealing function here, to remove the final step of removing the raw overheal from the effective healing of the trinket.
```

Comment:
I follow now.  Sorry, I had to also see it visually for myself to grasp it.  My assumption was more from the perspective that if we looked at effective healing, potentially the Velen's increase could be just the 15% there.  But your explanation clears that up.  I will revert the change to this,